src/test/logging_tests.cpp

```diff
@@ -114,7 +115,8 @@ BOOST_FIXTURE_TEST_CASE(logging_LogPrintMacros, LogSetup)
         "[net:debug] foo7: bar7",
         "[net:info] foo8: bar8",
         "[net:warning] foo9: bar9",
-        "[net:error] foo10: bar10"};
+        "[net:error] foo10: bar10",
+        "[validation] foo11: bar11"};
```

Comment:
```suggestion
        "[validation] foo11: bar11",
        };
```

nit, to make future changes less verbose

Reply:
thanks, done (we may need to update our clang-format)